.github/workflows/release.yaml

```
@@ -0,0 +1,115 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+jobs:
+  # From the tag we split the individual version components:
+  # major.minor.patch.stage<N>.
+  version:
+    name: Apply version
+    runs-on: ubuntu-latest
+    steps:
+      - name: split
+        id: split
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/}
+          echo $VERSION
+          PARTS=(${VERSION//\./ })
+          MAJOR=${PARTS[0]}
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+          echo "major=${MAJOR}" >> $GITHUB_OUTPUT
+          echo "minor=${PARTS[1]}" >> $GITHUB_OUTPUT
+          PARTS2=( $(grep -Eo '[[:digit:]]+|[^[:digit:]]+' <<< ${PARTS[2]} ) )
+          echo "patch=${PARTS2[0]}" >> $GITHUB_OUTPUT
+          echo "stage=${PARTS2[1]}" >> $GITHUB_OUTPUT
+          echo "build=${PARTS2[2]}" >> $GITHUB_OUTPUT
+
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get branch on which the tag is based
+        id: branchname
+        run: |
+          # get the branch on which the tag is based
+          raw=$(git branch -r --contains ${{ github.ref }})
```

The contains returns a list, better to use `--points-at` else a wrong branch might be selected.
```suggestion
          raw=$(git branch -r --points-at ${{ github.ref }})
```

---

I updated this on the `release/5.0` as wel.

---

We should check this also for other algorithms and for the algorithm template repo. Any other places you can think of?
Let's add an item to the sprint to make sure we don't forget this

---

https://github.com/orgs/vantage6/projects/1/views/53?pane=issue&itemId=103457388 - please check if my list is complete

---

Done